app.json

```diff
@@ -432,4 +440,4 @@
   },
   "success_url": "/",
   "website": "https://github.com/mitodl/mitxpro"
-}
+}
```

Comment:
nit: newline

Reply:
`app.json` is generated by a management command, changing it manually will break the Travis check so I can't edit this.